spark/v3.5/spark-runtime/LICENSE

```diff
@@ -519,30 +419,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains Carrot Search Labs HPPC.
-
-Copyright: 2002-2019 Carrot Search s.c.
-Home page: http://labs.carrotsearch.com/hppc.html
-License: http://www.apache.org/licenses/LICENSE-2.0
-
---------------------------------------------------------------------------------
-
-This binary artifact contains code from Apache Lucene via Carrot Search HPPC.
```

Comment:
I don't find lucene in the spark-runtime jar.

Reply:
It is referring to the above license entry, but since we don't have HPPC this makes sense to pull as well